extensions/ql-vscode/test/tsconfig.json

```diff
@@ -1,13 +1,11 @@
 {
   "extends": "../tsconfig.json",
-  "include": [
-    "**/*.ts",
-    "../src/vscode-tests/factories/db-config-factories.ts"
-  ],
+  "include": ["**/*.ts", "../src/**/*.ts"],
   "exclude": [],
   "compilerOptions": {
     "noEmit": true,
     "resolveJsonModule": true,
-    "rootDir": ".."
+    "rootDir": "..",
+    "jsx": "react"
```

Comment:
Why did you add this here?

Reply:
Because we now need to compile jsx files as part of `/test` (because we compile all of `/extensions/ql-vscode` which includes `/src`)